src/EnergyPlus/DXCoils.cc

```diff
@@ -2979,895 +2979,978 @@ void GetDXCoils(EnergyPlusData &state)
 
     // Loop over the Pumped DX Water Heater Coils and get & load the data
     CurrentModuleObject = HVAC::cAllCoilTypes(HVAC::CoilDX_HeatPumpWaterHeaterPumped);
-    for (DXHPWaterHeaterCoilNum = 1; DXHPWaterHeaterCoilNum <= state.dataDXCoils->NumDXHeatPumpWaterHeaterPumpedCoils; ++DXHPWaterHeaterCoilNum) {
+    auto &s_ip = state.dataInputProcessing->inputProcessor;
```

Comment:
I have a giant coil branch in the hopper and there are going to be a lot of conflicts. Just a matter of which one of us will have to fix them. 😄